samples/huawei-Kafka-autofill.yml

```diff
@@ -103,6 +105,7 @@ flavors:
   modificationImpact:
     isDataLost: true
     isServiceInterrupted: true
+  isDowngradeAllowed: false
```

Comment:
default value would be true. So it is better to have maximum examples with "true" value.